adoc-mode.el

```diff
@@ -248,6 +257,56 @@ See for example `tempo-template-adoc-title-1'."
                  (const :tag "tempo-snippets" tempo-snippets))
   :group 'adoc)
 
+(defcustom adoc-fontify-code-blocks-natively nil
+  "When non-nil, fontify code in code blocks using the native major mode.
+This only works for code blocks where the language is
+specified where we can automatically determine the appropriate
+mode to use.  The language to mode mapping may be customized by
+setting the variable `adoc-code-lang-modes'."
+  :group 'adoc
+  :type 'boolean
+  :safe 'booleanp)
```

Comment:
We should also get in the habit of adding `:package-version` to newly added stuff, so it's clear when they were added - in this case `:package-version '(adoc-mode . "0.7.0")`.

Reply:
Hm, if you say so. In that case there should actually be branch `release/0.7.0` where development branches like `SrcBlock/Org` would branch-off from. First the development branches would be merged into `release/0.7.0` and when everything is ready for release that stuff would be merged into `master` and tagged with `0.7.0`.

Or, maybe, we just have a rolling release.

Reply:
I did add `:package-version`.